### DISCUSSION TITLE
can't set `siteid` field value

### DISCUSSION THREAD

**Discussion by zubin999** [phalcon/cphalcon#15830] [Q&A]
I have a table:
![image](https://user-images.githubusercontent.com/9336471/146119414-b631ad67-4942-41ad-9be9-90a1828fc67c.png)

when insert a row:

![image](https://user-images.githubusercontent.com/9336471/146131901-f2e8f797-1505-4b7d-8373-20e84e32422e.png)

it returns true but the `siteid` in table is 0;

phalcon version: 3.4.5

**Comment by zubin999** [comment]
the follow is sql log:

```
INSERT INTO `xm_admin` (`userid`, `siteid`, `roleid`, `departmentid`, `username`, `nickname`, `name`, `sex`, `birthday`, `email`, `photo`, `qq`, `msn`, `telephone`, `mobile`, `address`, `zipcode`, `tips`, `created`, `createdby`, `updated`, `updatedby`, `disabled`, `password`, `stat`, `lastlogintime`, `lastloginip`, `voice`, `media_salt`, `is_show`, `introduce`) VALUES (?, DEFAULT, ?, ?, ?, ?, ?, DEFAULT, DEFAULT, DEFAULT, ?, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, ?, ?, ?, ?, ?, ?, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT) [3,1,1,"aa","qd","qd","http:\/\/****\/images\/default-head-2x.png",1639539292,1,1639539292,1,0,"b95c8daca76578372a6958e348f44cdd"]
```

**Comment by quasipickle** [comment]
Are the other fields properly populated?  Is it _just_ `siteid` that's not?

What response do you get from the database when you try to manually run that query?

Not having a primary key in the table _might_ be causing an issue.
